.github/workflows/install-dependencies.sh

```diff
@@ -56,7 +56,7 @@ sudo ln -s $HOME/go/bin/addlicense /bin
 sudo apt install -y build-essential
 
 # install skaffold
-curl -Lo skaffold https://storage.googleapis.com/skaffold/releases/v2.0.0/skaffold-linux-amd64 && \
+curl -Lo skaffold https://storage.googleapis.com/skaffold/releases/latest/skaffold-linux-amd64 && \
```

Comment:
Praise: Good call. Let's stick to `latest`.

Reply:
thank you for checking the latest binaries!